scripts/github_api

```diff
@@ -174,6 +210,42 @@ def query_pull_request(repo, number, **kwargs):
     return {k:getattr(pr, k) for k, v in kwargs.items() if v}
 
 
+def query_status(repo, reference, context, **kwargs):
+    """
+    Handler for the query_status command. Returns information about
+    the status of a GitHub issue or pull-request.
+
+    Args:
+        repo (str): target repository path. Format:
+            <repo owner>/<repo>
+        reference (str): git reference. Either a sha1 or a
+            pull-request number
+        context (str): status context to query
```

Comment:
Yes. We currently define this in the Jenkins GitHub pull request builder plugin. This is used to identify which status is being queried, e.g. to get the status of a Pylint validation